src/styles.css

```diff
@@ -16,8 +17,34 @@
 	html.dark {
 		color-scheme: dark;
 	}
+  
+	/** Scroll bar */
+
+	/**
+	 * `html`
+	 * Applies colors to the main *native* browser scrollbar.
+	 *
+	 * `html.firefox .custom-scrollbar`
+	 * On firefox all scrollbars are native. This styles them with our custom colors.
+	*/
+	html,
```

Comment:
@titoBouzout we should leave the main browser window scrollbars default. Custom styling for scrollbars should only be only the in-page content 
```suggestion
```

Reply:
Is not a custom scrollbar, its a native scrollbar styled

Reply:
It also themes scrollbars on firefox, which doesnt allow custom scrollbars

Reply:
Sorry I more just mean default vs non-default. I don't think this is a big issue or anything so feel free to keep it. It was just push back I got on the custom scrollbars before and the compromise was to only alter the in-page scrollbars and keep the main page ones the browser default

Reply:
No worries, the main issue is replacing the native scrollbars, as long as the main scrollbar keeps being native is a non-issue, and I believe the styled scrollbar looks better overall (while still being native)